test/run

```diff
@@ -691,5 +692,30 @@ function run_upgrade_test() {
   echo "  Upgrade tests succeeded!"
   echo
 }
+
+run_plugin_test() {
+  echo "  Testing plugin installation"
+  local test_name="plugin_install"
+
+  create_container \
+    "${test_name}" \
+    --env MYSQL_ROOT_PASSWORD=rootpass \
+    --env MYSQL_USER=user \
+    --env MYSQL_PASSWORD=foo \
+    --env MYSQL_DATABASE=db
+
+  test_connection "${test_name}" root rootpass
+  ip=$(ct_get_cip "${test_name}")
+
+  mysql_cmd "$ip" "root" "rootpass" -e "INSTALL PLUGIN SQL_ERROR_LOG SONAME 'sql_errlog' \G"
```

Comment:
SQL_ERROR_LOG is the plugin or SONAME? I do not follow.
It is not clear to me as a database dummy folk. Is it documented somewhere?

If not, can you please add this to documentation?